src/test/deadlock_tests/suite.h

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) 2019 Greg Griffith
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include "sync.h"
+
+#include "test/test_bitcoin.h"
```

Comment:
These includes are not used

Reply:
sync.h provides the tests with all of the locking macros being tested
test_bitcoin.h is unused, i will remove this

Reply:
I go by "include what you need", not "include what others need", but that's probably a matter of preference :-)